aspnet/fundamentals/environments.rst

```diff
@@ -72,6 +72,24 @@ For example, you can use the following code in you Configure method to setup env
 
 If the app is running in a ``Development`` environment, then it enables BrowserLink and development specific error pages (which typically should not be run in production). Otherwise, if the app is not running in a development environment, a standard error handling page is configured to be displayed in response to any unhandled exceptions.
 
+You may need to determine which content to send to the client at runtime, depending on the current environment. A common scenario when this happens is when a developer uses full scripts and style sheets for development and debugging vs. their minified counterparts for staging or production. You can do this by setting the names attribute of the ``<environment>`` Tag Helper that corresponds to the environment you want it to work in - ``Development``, ``Staging`` or ``Production``. 
```

Comment:
In a development environment, you generally serve non-minimized scripts and style sheets, which makes debugging easier. Production and test environments should be served the minified versions and generally from a CDN. You can ...